src/pymovements/utils/__init__.py

```diff
@@ -33,6 +33,7 @@
     pymovements.utils.paths
```

Comment:
please add the new module to the toctree here in the docstring, otherwise it won't show up in the documentation

Reply:
done